### PR TITLE
Remove TensorFlow dependency in `Reduce on Plateau Learning Rate Scheduler` example

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
This PR removes `TensorFlow` as a dependency in `examples/contrib/reduce_on_plateau.ipynb`, and uses only TFDS along with [grain](https://github.com/google/grain) for loading and preparing the dataset. 

Ref #656

### CODE REVIEW

**Review by rdyro — APPROVED** [review]
Thank you!
